Model/Invoice/StatusUpdater.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * Youama_OTP
+ *
+ * @author  David Belicza <87.bdavid@gmail.com>
```

Comment:
In any file what you created, you can add your own name, real one or nickname and your own email address if you want, instead of mine because you are the author of those files. :) (The other parts like license should be the same as it was before.)

You can append your name to the other files what you just modified, it is valid in PHPDoc:
```
 @author  David Belicza <87.bdavid@gmail.com>
 @author  Other Author <address@email.tld>
```